### DISCUSSION TITLE
Visibilité de la possibilité de se désinscrire

### DISCUSSION THREAD

**Discussion by MagaliMoserNoriaIRL** [unistra/esup-immersup-demandes#2] [5- Inscriptions aux créneaux d'immersion]
Les inscrits viennent beaucoup vers nous par mail pour demander à se désinscrire d'un créneau d'immersion, serait-il possible de rendre cette possibilité plus visible dans le cas où ils se connectent sur leur téléphone ?

**Comment by efinkbeiner5** [comment]
Où imagines-tu ce bouton ?

**Reply by MagaliMoserNoriaIRL** [reply]
Peut-être le placer en premier, tout à gauche, mais je ne sais pas si c'est vraiment approprié.

**Comment by efinkbeiner5** [comment]
Solution intermédiaire : Placer en première colonne le bouton
Solution Envisagée : Utiliser des cards au lieu du tableau pour la vue lycéen, optimisé sur téléphone
Chiffrage : Modéré

**Comment by efinkbeiner5** [comment]
Revue complète de "Mes immersions" en version 4
